runner/tests/tests.rs

```diff
@@ -13,6 +13,30 @@ fn test_choi_jamiolkowski_isomorphism() -> Result<(), String> {
     run_bitcode(bitcode, None)
 }
 
+#[test]
+fn test_full_qir_simple() -> Result<(), String> {
+    let bitcode = include_bytes!("resources/full-qir.bc");
+    run_bitcode(bitcode, Some("QIR_App_Test__Simple"))
```

Comment:
What do the strings (e.g. 'QIR_App_Test__Simple`) here represent?

Reply:
These strings are the names of the entry point function to execute. It's an optional parameter that is only required if there is more than one entry point function in the QIR program.